app/over_react_redux/todo_client/lib/src/components/app_bar/app_bar_local_storage_menu.dart

```diff
@@ -34,6 +34,9 @@ class AppBarLocalStorageMenuComponent extends UiComponent2<AppBarLocalStorageMen
   final _overlayRef = createRef<MenuOverlayComponent>();
   bool get _currentStateKeyIsReadOnly => TodoAppLocalStorage.reservedStateKeys.contains(props.currentDataSetName);
 
+  @override
+  get consumedProps => propsMeta.allExceptForMixins({MenuOverlayProps});
```

Comment:
This component should have had `consumedProps` before; typically a component wouldn't be updated like this.